### ISSUE TITLE
Not able to install properly using Pip

### ISSUE THREAD

**Issue by yogi86** [YelpArchive/sqlite3dbm#10]
Hi,
I am trying to install this with pip but get the follwing error

![image](https://cloud.githubusercontent.com/assets/14844130/10129348/beea6040-6582-11e5-9391-568a317b5c8e.png)

But, I think this type of error wasn't there before as I was able to use it, though it was with Python 2.7. I am trying to use it with Python 3.4

**Comment by coyotemarin** [comment]
Yeah, I'm pretty sure this is because it doesn't support Python 3, so, duplicate of #11.
